From patches Mon Sep 17 00:00:00 2001
From: Stefan Weil <sw@weilnetz.de>
To: qemu-trivial@nongnu.org, Stefan Hajnoczi <stefanha@gmail.com>,
 Stefan Weil <sw@weilnetz.de>
Cc: qemu-trivial@nongnu.org, Paolo Bonzini <pbonzini@redhat.com>,
 qemu-devel@nongnu.org, Stefan Weil <sw@weilnetz.de>
Date: Sun, 14 Apr 2013 22:52:50 +0200
Message-id: 1365972770-13777-1-git-send-email-sw@weilnetz.de
Subject: [PATCH 1/1] hw: Add lost ARM core again

Commit 0434e30afb6175212389811e0b28b948eb3c1e40 accidentally removed
the compilation of arm11mpcore.c.

Signed-off-by: Stefan Weil <sw@weilnetz.de>
---
 default-configs/arm-softmmu.mak |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/default-configs/arm-softmmu.mak b/default-configs/arm-softmmu.mak
index 31725a9..27cbe3d 100644
--- a/default-configs/arm-softmmu.mak
+++ b/default-configs/arm-softmmu.mak
@@ -34,8 +34,8 @@ CONFIG_PFLASH_CFI02=y
 CONFIG_MICRODRIVE=y
 CONFIG_USB_MUSB=y
 
-CONFIG_ARM5MPCORE=y
 CONFIG_ARM9MPCORE=y
+CONFIG_ARM11MPCORE=y
 CONFIG_ARM15MPCORE=y
 
 CONFIG_ARM_GIC=y
-- 
1.7.10.4


